.github/CODEOWNERS

```diff
@@ -148,6 +148,8 @@
 /stdlib/public/Cxx/                       @zoecarver @hyp @egorzhdan
 /stdlib/public/Distributed/               @ktoso
 /stdlib/public/Observation/               @phausler
+/stdlib/public/runtime/                   @mikeash @al45tair
+/stdlib/public/Threading/                 @al45tair
 /stdlib/public/Windows/                   @compnerd
 /stdlib/public/libexec/swift-backtrace/   @al45tair
```

Comment:
Please sort these in case-sensitive order.

Reply:
Done.